components/report-project-dialog.tsx

```
@@ -0,0 +1,307 @@
+
+      // Success - close dialog and refresh
+      onOpenChange(false)
+      router.refresh()
+      
+      // Reset form
+      setReason('')
+      setDescription('')
+      setEvidenceFiles([])
+      evidencePreviews.forEach(url => URL.revokeObjectURL(url))
+      setEvidencePreviews([])
+    } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)